src/neptune/internal/utils/dependency_tracking.py

```diff
@@ -30,7 +30,7 @@ def log_dependencies(self, run: "Run") -> None:
 class InferDependenciesStrategy(DependencyTrackingStrategy):
     def log_dependencies(self, run: "Run") -> None:
         try:
-            dependencies_str = subprocess.check_output(["pipreqs", "--print", "."]).decode("utf-8")
+            dependencies_str = subprocess.check_output(["pip", "freeze"]).decode("utf-8")
```

Comment:
@AleksanderWWW `pip list --format=freeze` is a more reliable option as it correctly parses requirements in a conda environment
https://stackoverflow.com/questions/62863020/pip-freeze-generating-file-on-conda-environment

Reply:
I see in the comments on SO that there are some issues with it. For conda, we would probably go with a dedicated integration.

Reply:
@AleksanderWWW tested and said it works fine
There's a new PR: https://github.com/neptune-ai/neptune-client/pull/1389/files#diff-8d10a7f1e52cebb61d979c1afe224d21119c3f42b698cd2252ab88ab72598c72R35

Reply:
The comment there says that if someone installs a dependency from git then it will not be shown in the output. Is that a deal-breaker for us @twolodzko @SiddhantSadangi?

Reply:
It worked for me using `pip==23.1.2`